docs/oracle/tech-architecture.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Technical Architecture
 This document explains the general architecture of UMA's DVM (Oracle). It is meant to be a high level overview and is
 by no means an exhaustive explanation of the system.
 
-![Diagram](/docs/oracle/architecture_diagram.jpeg)
+![Diagram](/docs/oracle/OO-architecture-diagram-v2.png)
```

Comment:
This looks a lot better design wise, but is still outdated content. Is this the right version? I don't want to block this PR to update this, but we should make a not for the future because this does not include the optimistic oracle.

Reply:
Yea, I updated this for what Jesper created and then I had in my notes we were going to use a diagram that Reinis made and forgot to follow up. Let's just include the update in this PR and I will update in a separate PR after following up with Reinis.

Reply:
Sounds good.